### PR TITLE
[SCB-188] Transport optimize for reactive

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

File: transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayConfig.java [hunk 1]
```diff
@@ -42,10 +42,4 @@ public static int getClientThreadCount() {
         DynamicPropertyFactory.getInstance().getIntProperty("cse.highway.client.thread-count", 1);
     return address.get();
   }
-
-  public static int getClientConnectionPoolPerThread() {
-    DynamicIntProperty address =
-        DynamicPropertyFactory.getInstance().getIntProperty("cse.highway.client.connection-pool-per-thread", 1);
```
Comment (liubao68) [inline transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayConfig.java]: documents should modified accordingly

File: demo/perf/src/main/resources/microservice.yaml [hunk 2]
```diff
@@ -15,28 +15,47 @@
 ## limitations under the License.
```
Comment (liubao68) [inline demo/perf/src/main/resources/microservice.yaml]: META-INF  LICENCE.txt & NOTICE.txt files are missing

Comment (wujimin) [reply demo/perf/src/main/resources/microservice.yaml]: done

File: transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayConfig.java (hunk 1 above)
Comment (wujimin) [reply transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayConfig.java]: done

File: demo/perf/src/main/resources/microservice.yaml (hunk 2 above)
Comment (WillemJiang) [reply demo/perf/src/main/resources/microservice.yaml]: Hi, you don't need add the LICENCE.txt & NOTICE.txt by hand any more.  There is a maven plugin which could take care of these license files when building the artifacts.

File: demo/perf/pom.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (WillemJiang) [inline demo/perf/pom.xml]: Please add the license header here.

File: demo/perf/src/main/java/io/servicecomb/demo/perf/Intf.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package io.servicecomb.demo.perf;
```
Comment (WillemJiang) [inline demo/perf/src/main/java/io/servicecomb/demo/perf/Intf.java]: License header.

File: demo/perf/src/main/java/io/servicecomb/demo/perf/PerfMain.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (WillemJiang) [inline demo/perf/src/main/java/io/servicecomb/demo/perf/PerfMain.java]: Please update the license header.

File: demo/perf/pom.xml (hunk 3 above)
Comment (wujimin) [reply demo/perf/pom.xml]: done

File: demo/perf/src/main/java/io/servicecomb/demo/perf/Intf.java (hunk 4 above)
Comment (wujimin) [reply demo/perf/src/main/java/io/servicecomb/demo/perf/Intf.java]: done

File: demo/perf/src/main/resources/microservice.yaml (hunk 2 above)
Comment (wujimin) [reply demo/perf/src/main/resources/microservice.yaml]: deleted

File: demo/perf/src/main/java/io/servicecomb/demo/perf/PerfMain.java (hunk 5 above)
Comment (wujimin) [reply demo/perf/src/main/java/io/servicecomb/demo/perf/PerfMain.java]: done

**Pull request merged**